src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -21,35 +21,35 @@ You can customize the creation of a new [BrowserContext](/javascript-api/xk6-bro
+| javaScriptEnabled                   | boolean | `true`          | Whether to activate JavaScript support for the context.                                                                                                                                                                                           |
+| locale                              | string  | system          | Specifies the user's locale, such as `en-US`, `tr-TR`, etc.                                                                                                                                                                                       |
+| offline                             | boolean | `false`         | Whether to emulate an offline network.                                                                                                                                                                                                            |
+| permissions                         | Array   | `null`          | Permissions to grant for the context's pages. See [browserContext.grantPermissions()](/javascript-api/xk6-browser/browsercontext#browsercontext-grantpermissions-permissions-options) for the options.                                            |
+| reducedMotion                       | string  | `no-preference` | Minimizes the amount of motion by emulating the 'prefers-reduced-motion' media feature. It can be one of `'reduce'` and `'no-preference'`. See [page.emulateMedia()](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options. |
+| screen                              | object  | `null`          | Sets a window screen size for all pages in the context. It can only be used when the viewport is set. See: [screen](#screen-options) for the options.                                                                                             |
+| timezoneID                          | string  | system          | Changes the context's timezone. See [ICU's metaZones.txt](https://cs.chromium.org/chromium/src/third_party/icu/source/data/misc/metaZones.txt?rcl=faee8bc70570192d82d2978a71e2a615788597d1) for a list of supported timezone IDs.                 |
+| userAgent                           | string  | browser         | Specifies the user agent to use in the context.                                                                                                                                                                                                   |
+| viewport                            | object  | `1280x720`      | Sets a viewport size for all pages in the context. `null` disables the default viewport. See: [viewport](#viewport-options) for the options.                                                                                                      |
```

Comment:
To be precise:

```suggestion
| viewport                            | object  | `{width: 1280, height: 720}`      | Sets a viewport size for all pages in the context. `null` disables the default viewport. See: [viewport](#viewport-options) for the options.                                                                                                      |
```

Otherwise `1280x720` might confuse someone that a string is acceptable (which wouldn't be a bad feature to have).

Same for other instances.

Reply:
I've committed the change in `0438d60d0bf3c3342426fc760c771c330524b022` as i couldn't commit this suggestion after the other commits 🤷